test/integration/user_visits_place_test.rb

```diff
@@ -8,12 +8,12 @@ class UserVisitsPlaceSpec < ActionDispatch::IntegrationTest
     VCR.use_cassette("new place") do
       visit "/"
 
-      page.fill_in 'nav-search', 
+      page.fill_in 'nav-search',
         :with => 'Turing School of Software & Design'
 
       click_button "Go"
 
-      assert_equal '/places/turing-school-of-software-design', 
+      assert_equal '/places/turing-school-of-software-design',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.